src/libraries/Microsoft.PowerFx.Core/Public/CheckResult.cs

```diff
@@ -715,9 +715,9 @@ public CheckContextSummary ApplyGetContextSummary()
             return summary;
         }
 
-        public IEnumerable<string> GetFunctionNames()
+        public IEnumerable<string> GetFunctionNames(bool annonymizeUnknownPublicFunctions = false)

```

Comment:
>(bool annonymizeUnknownPublicFunctions = false) [](http://example.com/codeflow?start=51&length=47)

Can we avoid a binary breaking change and chain it to a new function definition?

Reply:
I dont think this is necessary in this case.

Reply:
It actually does help with the some deployments(e.g. Dataverse) where they can dynamically load new nugets without re-compiling and this kind of binary breaking changes can cause issues.

Reply:
We should avoid the breaking change.